packages/server/templates/big_badge.ejs

```diff
@@ -0,0 +1,52 @@
+<% 
```

Comment:
Add copyright notice, please. As I can see, this SVG code is copied from some badge.

Reply:
I made this badge in Figma, then exported it to SVG.

Reply:
we discussed the creation

Reply:
Ah, sorry, forgot about that